### PR TITLE
Java/Kotlin: Make the basic query in docs work for both languages

### PR DESCRIPTION
**Pull request by igfoo**

### CODE REVIEW

**Review by subatoi — APPROVED** [review]
Looks good to me, thank you, but tagging @felicitymay just in case. There are some older style adherences here and references to LGTM (which is being deprecated) but I think we're OK since this is the CodeQL microsite

File: docs/codeql/codeql-language-guides/basic-query-for-java-code.rst [hunk 1]
```diff
@@ -1,18 +1,33 @@
 .. _basic-query-for-java-code:
 
-Basic query for Java code
-=========================
+Basic query for Java and Kotlin code
```
Comment (felicitymay) [inline docs/codeql/codeql-language-guides/basic-query-for-java-code.rst:3]: Ideally we'd also update this article name in the "CodeQL for Java and Kotlin" article, but that will be easier to do once one of these PRs is merged.

File: docs/codeql/codeql-language-guides/basic-query-for-java-code.rst [hunk 2]
```diff
@@ -57,44 +75,43 @@ Running the query
...
+| Query part                                                                                       | Purpose                                                                                                           | Details                                                                                           |
++==================================================================================================+===================================================================================================================+===================================================================================================+
+| ``import java``                                                                                  | Imports the standard CodeQL libraries for Java.                                                                   | Every query begins with one or more ``import`` statements.                                        |
```
Comment (felicitymay) [inline docs/codeql/codeql-language-guides/basic-query-for-java-code.rst:100]:
```suggestion
| ``import java``                                                                                  | Imports the standard CodeQL libraries for Java and Kotlin.                                                                   | Every query begins with one or more ``import`` statements.                                        |
```

**Review by felicitymay — APPROVED** [review]
Many thanks for the updates @igfoo 💖 

I found a couple of minor things to comment on, but otherwise this looks good.

**Comment by igfoo** [comment]
I made the second change, but I think it'll be simpler to do the rename in the other PR

**Pull request merged**